test/TestProjects/FirstTest-Cadl/Generated/ExtensibleEnum.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace CadlFirstTest
+{
+    /// <summary> Extensible enum. </summary>
+    public enum ExtensibleEnum
```

Comment:
This is a fixed enum. We should rename it.

Reply:
It is extensible enum.